examples/how-to-use-webrtc-star/README.md

```diff
@@ -0,0 +1,14 @@
+# How to use WebRTC star
+
+There's still a bit of work required to start up an in-browser node in a robust way, so that it will work whether or not there is an existing initialized IPFS repo in the user's browser. If there isn't one, you need to call `init` as above, but if there is one, calling `init` will fail. Moreover, there's currently no good way to check if you need to call `init` or not.
```

Comment:
Not sure how that is related to "how to use webrtc-star", even if it's good information in general when using js-ipfs.